tests/typowalking/markAllTypos.js

```diff
@@ -0,0 +1,129 @@
+/* bender-tags: editor,unit */
+/* bender-ckeditor-plugins: nanospell */
+
+'use strict';
+
+bender.editor = {
+	config: {
+		enterMode: CKEDITOR.ENTER_P
+	}
+};
+
+bender.test( {
+	assertHtml: function( expected, actual, msg ) {
+		assert.areEqual( bender.tools.fixHtml( expected ), bender.tools.fixHtml( actual ), msg );
+	},
+	setupSpy: function() {
+		var editor = this.editorBot.editor,
+			markTyposSpy;
+
+		markTyposSpy = sinon.spy(editor.plugins.nanospell, 'markTypos');
```

Comment:
markAllTypos basically walks over the document to get each paragraph-like element, and passes it into markTypos.  So we set up this sinon spy to observe what gets passed in.